scripts/create_makefile.rb

```diff
@@ -58,7 +58,7 @@
   test_sources = Dir["#{TEST_DIR}/**/test_*.c"]
   test_targets = []
   generator = UnityTestRunnerGenerator.new
-  all_headers = Dir["#{SRC_DIR}/**/*.h"]
+  all_headers = Dir["#{SRC_DIR}/**/{[!mock_]}*.h"]  #headers that begin with mock_ are not included
```

Comment:
I know this is merged, but I think we have postfix as well - can we make this a bit more universal?

Reply:
true, it should not be hardcoded with `mock_` (default config) but it should use the `mock_prefix` and `mock_suffix` configuration variables

Reply:
Agreed. Can one of you post a PR to add that or should I?

Reply:
PR in https://github.com/ThrowTheSwitch/CMock/pull/145

Feel free to comment, this is my 2nd ruby experience :wink: